File: SDKs/Aspose.Pdf_Cloud_SDK_for_Java/src/main/java/com/aspose/pdf/model/DocumentPropertiesResponse.java
package com.aspose.pdf.model;

import com.aspose.pdf.model.DocumentProperties;
public class DocumentPropertiesResponse {
  private DocumentProperties DocumentProperties = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getDocumentProperties
	 * Gets DocumentProperties
	 * @return DocumentProperties
	 */
  public DocumentProperties getDocumentProperties() {
    return DocumentProperties;
  }

	/**
	 * setDocumentProperties
	 * Sets DocumentProperties
	 * @param DocumentProperties DocumentProperties
	 */
  public void setDocumentProperties(DocumentProperties DocumentProperties) {
    this.DocumentProperties = DocumentProperties;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentPropertiesResponse {\n");
    sb.append("  DocumentProperties: ").append(DocumentProperties).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/DocumentPropertiesResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class DocumentPropertiesResponse {
    public DocumentProperties DocumentProperties { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DocumentPropertiesResponse {\n");
      sb.Append("  DocumentProperties: ").Append(DocumentProperties).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
